.teamcity/tests/buildTypes/E2ETestsViaCompose.kt

```diff
@@ -0,0 +1,46 @@
+package tests.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.BuildType
+import jetbrains.buildServer.configs.kotlin.buildFeatures.PullRequests
+import jetbrains.buildServer.configs.kotlin.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+
+
+object E2ETestsViaCompose : BuildType({
```

Comment:
It seems like something can be improved here.

- Do we really need another different configuration? Maybe we can merge it with E2ETests?
- If it is still needed, I would prefer to see in the title what it does, not how :)

Reply:
✅